src/Particle/ParticleContainer.H

```diff
@@ -312,8 +312,34 @@ void make_ParticleContainer_and_Iterators (py::module &m, std::string allocstr)
         //                  const Vector<std::string>& int_comp_names = Vector<std::string>()) const
         // void CheckpointPre ();
         // void CheckpointPost ();
+
         // void Restart (const std::string& dir, const std::string& file);
+        .def("restart",
+            [](
+                ParticleContainerType & pc,
+                std::string const & dir,
+                std::string const & file
+            ){
+                return pc.Restart(dir, file);
+            },
+            py::arg("dir"),
+            py::arg("file")
+        )
+
         // void Restart (const std::string& dir, const std::string& file, bool is_checkpoint);
+        .def("restart_checkpoint",
+            [](
+                ParticleContainerType & pc,
+                std::string const & dir,
+                std::string const & file,
+                bool is_checkpoint
+            ){
+                return pc.Restart(dir, file, is_checkpoint);
+            },
```

Comment:
Potential shortcut to try:
```suggestion
            py::overload_cast<std::string const &, std::string const &, bool>(&ParticleContainerType::Restart),
```